script/configs/exclude_all_plugins_app.yaml

```diff
@@ -8,3 +8,4 @@
 
 # This is a permament entry, as it should never be a direct app dependency.
 - plugin_platform_interface
+- webview_flutter_platform_interface
```

Comment:
No need to update the PR and do CI again for this, but for the next round: we should put comments about why we're temporarily adding things here (with issue reference) even when we plan to almost immediately remove them again because it's easy to forget to take things back off of this list, so having a clear way to see what's an entry that we just forgot to remove is helpful.